### PR TITLE
Feat/rename hostname to host friendly name

### PR DESCRIPTION
**Pull request by NatLee**
Renaming the variables in the code to improve readability.
Thank @h-alice for inspiring me.

I pull https://github.com/NatLee/telepy/pull/27 in the first, so it may show some codes with commits of the first login HTML. :(

### CODE REVIEW

File: src/authorized_keys/admin.py [hunk 1]
```diff
@@ -47,8 +47,8 @@ def clean_key(self):
 @admin.register(ReverseServerAuthorizedKeys)
 class ReverseServerAuthorizedKeysAdmin(admin.ModelAdmin):
     form = ReverseServerAuthorizedKeysForm
-    list_display = ('user', 'hostname', 'display_key', 'reverse_port', 'created_at', 'updated_at')
-    search_fields = ('user', 'hostname', 'key', 'reverse_port', 'description')
+    list_display = ('user', 'host_friendly_name', 'display_key', 'reverse_port', 'created_at', 'updated_at')
```
Comment (h-alice) [inline src/authorized_keys/admin.py:50]: Friendly! : D

Comment (NatLee) [reply src/authorized_keys/admin.py:50]: Friendly friend on Friday! :D

**Comment by NatLee** [comment]
@h-alice please review again, thanks! :3

File: src/static/js/helper.js [hunk 2]
```diff
@@ -76,6 +76,20 @@ function isValidSSHKey(key) {
    return true;
  }
 
+function getHostFriendlyNameFromKey(key, hostFriendlyName) {
```
Comment (h-alice) [inline src/static/js/helper.js:79]: Friendly! : D

File: src/static/js/keys.js [hunk 3]
```diff
@@ -32,7 +32,7 @@ function fetchAndDisplayUserKeys() {
             `;
 
             const row = `
-            <tr onclick="showKeyDetails('${hostFriendlyName}', '${publicKey}', ${itemDescription})">
+            <tr onclick="showKeyDetails('${hostFriendlyName}', '${publicKey}', '${itemDescription}')">
```
Comment (h-alice) [inline src/static/js/keys.js:35]: It seems a small fix of HTML element content.

File: src/static/js/helper.js (hunk 2 above)
Comment (NatLee) [reply src/static/js/helper.js:79]: Friendly friend! : D

File: src/static/js/keys.js (hunk 3 above)
Comment (NatLee) [reply src/static/js/keys.js:35]: Yap. We just lost `'` 🥲

File: src/static/js/helper.js (hunk 2 above)
Comment (h-alice) [reply src/static/js/helper.js:79]: But fk JavaScript

**Review by h-alice — APPROVED** [review]
It seems some small change of front-end part.

Everything is OK and ready to be merged.

File: src/static/js/helper.js (hunk 2 above)
Comment (NatLee) [reply src/static/js/helper.js:79]: No typing language will die out sooner or later. :D

**Pull request merged**